website/docs/intro/index.mdx

```diff
@@ -16,7 +16,7 @@ Terraform creates and manages resources on cloud platforms and other services th
 
 ![Terraform creates and manages cloud platforms and services through their APIs](/img/docs/intro-terraform-apis.png)
 
-HashiCorp and the Terraform community have already written **more than 1700 providers** to manage thousands of different types of resources and services, and this number continues to grow. You can find all publicly available providers on the [Terraform Registry](https://registry.terraform.io/), including Amazon Web Services (AWS), Azure, Google Cloud Platform (GCP), Kubernetes, Helm, GitHub, Splunk, DataDog, and many more.
+HashiCorp and the Terraform community have already written **more than 2400 providers** to manage thousands of different types of resources and services, and this number continues to grow. You can find all publicly available providers on the [Terraform Registry](https://registry.terraform.io/), including Amazon Web Services (AWS), Azure, Google Cloud Platform (GCP), Kubernetes, Helm, GitHub, Splunk, DataDog, and many more.
```

Comment:
```suggestion
HashiCorp and the Terraform community have already written **more than 2500 providers** to manage thousands of different types of resources and services, and this number continues to grow. You can find all publicly available providers on the [Terraform Registry](https://registry.terraform.io/), including Amazon Web Services (AWS), Azure, Google Cloud Platform (GCP), Kubernetes, Helm, GitHub, Splunk, DataDog, and many more.
```

Reply:
@brandonc it's even higher now! :)

Reply:
okay I like this - it's definitely correct and definitely more evergreen. Please add this in and then merge! P.S. I'm adding a backport label to your PR. After you merge this, please also merge the auto-generated backport label. This will make sure this change is reflected in our most recent docs version!